examples/deploy_tasks.yaml

```diff
@@ -6,9 +6,7 @@ metadata:
 spec:
     inputs:
         resources:
-            - resourceRef:
-                name: resource-name
-              name: workspace
+              - name: workspace
```

Comment:
i think this should have a type as well, in this case `git` or `github`, so that a person using the Task knows what kind of resource to provide

Reply:
will add type, makes sense for clarity